windows/Flutter Windows Embedder.sln

```diff
@@ -0,0 +1,44 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.27703.2026
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "GLFW Library", "GLFW Library\GLFW Library.vcxproj", "{90057FD8-9460-43A6-8CDF-3AAC1C4255E5}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "GLFW Example", "GLFW Example\GLFW Example.vcxproj", "{5A827760-CF8B-408A-99A3-B6C0AD2271E7}"
```

Comment:
Perhaps swap the order of the two projects so that GLFW Example is the default project on initial launch, making Run do something without needing to change settings?